package.json

```diff
@@ -6,7 +6,15 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "nodemon --exec babel-node src/index",
-    "start": "babel-node ./src"
+    "start": "babel-node ./src",
+    "sequelize": "node_modules/.bin/sequelize",
+    "db:migrate": "npm run sequelize db:migrate nomad",
+    "migrate": "npx sequelize-cli db:migrate",
```

Comment:
Why did you use `npx sequelize-cli` while on the above line you used `npm run sequelize`